src/codeu/chat/client/commandline/Panel.java

```diff
@@ -30,7 +33,46 @@
     void invoke(List<String> args);
   }
 
+  class TimerCommand {
```

Comment:
Add javadoc describing class.

Can this be a generic class in `codeu.chat.util`? Something like `Scheduled<? extends Action>` with `Command` implementing the `Action` interface.